### PR TITLE
Add back button to project config

### PR DESCRIPTION
**Pull request by jakic12**
It may not be the best, but it works :)

### CODE REVIEW

File: frontend/src/pages/start/configuration/Configuration.tsx [hunk 1]
```diff
@@ -194,6 +197,18 @@ const Configuration: FunctionComponent = () => {
...
   return (
     <div className={classes.root}>
+      <div
+        className={classNames(classes.backButtonWrapper, {
+          [classes.isConfig]: !!id,
+        })}
+      >
+        <IconBackButton
+          onClick={() => {
+            window.history.back();
```
Comment (bartolomej) [inline frontend/src/pages/start/configuration/Configuration.tsx:207]: Do you think it's better to use `history.back()` from `const history = useHistory()` hook.

See: https://v5.reactrouter.com/web/api/Hooks/usehistory.

Not sure if there are any technical differences between those two, but if nothing less it's for consistency sake.

File: frontend/src/pages/start/Start.tsx [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
-        <Route path={`/start/configure/:id?`} component={Configuration} />
+        <Route
+          path={`/start/configure`}
+          exact={true}
+          component={Configuration}
+        />
+        <RouteWithLayout
+          path={`/start/configure/:id?`}
+          component={Configuration}
+        />
```
Comment (bartolomej) [inline frontend/src/pages/start/Start.tsx:26]: Did you test if the project settings screen logic still works for both of these routes? I think this shouldn't break anything, but just asking to be sure.

File: frontend/src/pages/start/Start.tsx [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
+import { Redirect, Route, RouteProps, Switch } from "react-router-dom";
 import Main from "./main/Main";
 import Configuration from "./configuration/Configuration";
+import Layout from "components/layout/Layout";
+
+const RouteWithLayout = (props: RouteProps) => (
+  <Layout>
+    <Route {...props} />
+  </Layout>
+);
```
Comment (bartolomej) [inline frontend/src/pages/start/Start.tsx:11]: Nit: Could we reuse the `RouteWithLayout` component that's already defined in `App.tsx` (i think)?

**Review by bartolomej — APPROVED** [review]
Looks good!

File: frontend/src/pages/start/Start.tsx (hunk 2 above)
Comment (jakic12) [reply frontend/src/pages/start/Start.tsx:26]: yep, checked both

File: frontend/src/pages/start/Start.tsx (hunk 3 above)
Comment (jakic12) [reply frontend/src/pages/start/Start.tsx:11]: I did not want to use it, because this would result in a circular import. Will make it non redundant.

**Review by bartolomej — APPROVED** [review]

**Pull request merged**